main/boost/APKBUILD

```diff
@@ -131,14 +156,30 @@ _boostlib() {
 	esac
 
 	mkdir -p "$subpkgdir"/usr/lib
-	mv "$pkgdir"/usr/lib/libboost_$name* "$subpkgdir"/usr/lib/
+	# -dev might have moved all these files already
+	local _lib; for _lib in "$pkgdir"/usr/lib/libboost_$name* ; do
+		test ! -e "$_lib" || mv "$_lib" "$subpkgdir"/usr/lib/
+	done
+
+	rmdir "$pkgdir"/usr/lib >/dev/null 2>&1 || :
+}
+
+_boost_bundle() {
+	pkgdesc="Boost bundle meta-package"
+	depends="$pkgname"
+
+	local _pkg
+	for _pkg in ${origsubpackages% *}; do
+		depends="$depends ${_pkg%%:*}"
+	done
+
+	mkdir -p "$subpkgdir"
 }
 
 _pyversion() {
-	$1 -c 'import sys; print("%i.%i" % (sys.version_info.major, sys.version_info.minor))'
+	"$(command -v "$1" || echo :)" -c 'import sys; print("%i.%i" % (sys.version_info.major, sys.version_info.minor))'
```

Comment:
i dont think we want the ` ... || echo :` in there, as it will make us silently ignore errors. I think we want to know if we for some reason fail to detect python version.